src/domain_factory/account/depot_account.py

```diff
@@ -0,0 +1,28 @@
+from domain_factory.account.brokerage_account import BrokerageAccount
+
+
+class DepotAccount(BrokerageAccount):
+    """ Class to generate depot accounts. Generate method will call the
```

Comment:
Same as with ECP, we should be consistent with cases.  It's "depot accounts" here but "Depot accounts" in the generate method

Reply:
done